views/component/Page.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <base href="/">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>{% block title %}{% endblock %} - Civil Service Learning</title>
+    <link rel="stylesheet" href="/styles/main.css?v=">
```

Comment:
I've removed a lot of the previous imports from Page.html (e.g. Google Analytics), but will replace in future as and when we need